src/gamepad/controller.cpp

```diff
@@ -15,6 +20,29 @@ void Controller::updateButton(pros::controller_digital_e_t button_id) {
     }
 }
 
+void Controller::updateScreen() {
+    if (pros::millis() - this->last_print_time < 50)
+        return;
+
+    for (int i = 1; i < 3; i++) {
+        int line = (this->last_printed_line + i) % 3;
+        
+        if (pros::millis() - this->line_set_time[line] < this->screen_contents[line].second)
+            continue;
+        
+        
+        this->controller.clear_line(line);
+        this->controller.set_text(line, 0, this->screen_buffer[line][0].first.c_str());
```

Comment:
```suggestion
        this->controller.set_text(line + std::string(40, ' '), 0, this->screen_buffer[line][0].first.c_str());
```
Use extra spaces to clear line instead of using `clear_line` (see above).

Reply:
Note: The exact number of spaces will likely need some tuning.